### ISSUE TITLE
Second developer/maintainer needed

### ISSUE THREAD

**Issue by pszafer** [bosch-thermostat/home-assistant-bosch-custom-component#414]
Hi all,

I'd like to ask for help in developing this component. I don't want to abandon this project, there are many users of it and I don't want to abandon anyone.

The best would be the person with access to Easycontrol device.
I have access only to RC300 device; I, myself, don't own any Bosch device. It's getting harder to get things fixed.

I don't have enough free time to make it work smooth for all type of devices; 
I developed this component 6 years ago and in Home Assistant world is like century.

HA make changes too frequent lately for me to make it working after every update.
I can spare my time to teach somebody how I organized this project and if possible in time I'd like to make this project goes into Home Assistant Core as regular integration.

**Comment by justinh998** [comment]
Actually I would like to propose myself as a maintainer here but only later, at the moment I think that I would be rather unsuitable because 
1. I know very little programming .I'm just learning Python.
2. I really don't know anything about home assistant logic for plugins.
3. i own an easycontrol but that alone does nothing for me. 
I would rather see myself as a maintainer in 2-3 years.

**Comment by justinh998** [comment]
Actually I would like to propose myself as a maintainer here but only later, at the moment I think that I would be rather unsuitable because 
1. I know very little programming .I'm just learning Python.
2. I really don't know anything about home assistant logic for plugins.
3. i own an easycontrol but that alone does nothing for me. 
I would rather see myself as a maintainer in 2-3 years.

**Comment by Jojo10011** [comment]
I'd be interested in helping out.

- I've worked with python professionally for about 4 years at this point.
- I have worked with async code in the past
- I am studying Computer Science at a University currently
- I have access to a Buderus Logamatic TC100.2 which is Easycontrol if i remember correctly. I also have a KM 200 which is currently not in use.

I have however not worked with Homeassistant and I dont have a lot of experience with open source work. So there would be a bit of a learning curve

**Comment by pszafer** [comment]
@Jojo10011 if you're interested in contributing, join our discord and PM me. I can help you dive into HA dev world.

**Comment by Jojo10011** [comment]
@pszafer I just sent you a PM.  look for DawnOfJojo on Discord. I‘ve been in the discord since 2022

**Comment by rikroe** [comment]
I can also help out on the Home Assistant side (currently maintaining the HA BMW integration).
Have a CT200 so will look if I am at least able to review/test some PRs!

**Comment by enric0r** [comment]
@pszafer I'll be more than happy to help out as well.

**Comment by avalanched** [comment]
I can help as well.
developer since 2008, but only just a newbie to HA.
i have a buderus TC100.2

**Comment by omerfaruk-aran** [comment]
Hi @pszafer,

First, thank you for maintaining this project for 6 years and for all your contributions to the Home Assistant community. I understand how important this project is to many users, and I’d love to help ensure its future development.  

Let me introduce myself: I am a developer from Turkey with a strong interest in Home Assistant and smart home projects. I’d like to be transparent—I don’t speak English fluently, but I’m eager to contribute and will do my best to communicate effectively.  

I currently own a Buderus GB122i boiler and a TC100.2 thermostat, which gives me the ability to develop and test the integration directly. Additionally, I’ve been working on other Home Assistant and ESPHome projects, such as integrating Samsung HVAC systems with Wi-Fi and RS485 protocols. You can view my related projects here:  
- [GitHub Profile](https://github.com/omerfaruk-aran)  
- [ESPHome Samsung HVAC Bus Integration](https://github.com/omerfaruk-aran/esphome_samsung_hvac_bus)  

I recently completed a smart kettle integration project as well, showcasing my dedication to building practical solutions for Home Assistant users.  

I’m very interested in taking over your project and ensuring that it continues to benefit the community. I would also be happy to learn from you how the project is organized and maintained.  

Looking forward to hearing from you and potentially collaborating.  

Best regards,  
Omer Faruk Aran  
[GitHub Profile](https://github.com/omerfaruk-aran)

**Comment by totyaone** [comment]
Hi @pszafer,

I own a bosh CT200(firmeware 05.02.00) with three thermostat and will be add another three soon.   I'm new in HA but more then 20 years develpoing in perl,php, js,css and using linux so with a few guidance I can make test, discover, debug scan if needed.

**Comment by pszafer** [comment]
Hi @avalanched @omerfaruk-aran @totyaone you're welcome to join discord, pm and we can work together.
I think for now PR's would with fixes and new features would be nice as there were plenty people who wanted to help and after they see how complicated Bosch ecosystem is they changed their minds

**Comment by Bigglesuk** [comment]
I'm a developer (of sorts), having used HA for many years now, but your integration is one of my key reasons for going that way. I'd like to help if I can. I've never done development of HA, but I write integrations between my ERP system and various other products. I also have the CT200 and 12 radiator thermostats. I shall look to join your discord.

**Comment by Salvora** [comment]
is the project on hiatus right now?

**Comment by HuisAutomatisering** [comment]
I don't know how to connect to developers but I will try. https://github.com/Martijn-W/dev.martijn.easycontrol
@Martijn-W

**Comment by FredAloneForEver** [comment]
> Hello,
> 
> I'm not a developer, but I use Home Assistant extensively and have a Junkers CT100. Your integration already works quite well for the basics, but I would like to see it expanded with additional features. Unfortunately, I can't program myself, but I would be happy to provide you with any data you need from my CT100. I am particularly interested in functions such as enabling/disabling hot water and scheduling heating and hot water times.

Yes I would like to these features (Hot water on/of, scheduling ...)
